cmd/crdvalidator/README.md

```diff
@@ -0,0 +1,116 @@
+# crdvalidator
+
+## Summary
+A part of the core value proposition for RukPak is the safe upgrade of `CRD`s. As a result, this repository comes equipped with a "CRD Validator Webhook", `crdvalidator`, that will validate specified `CRD` upgrade events.
```

Comment:
I think it would also be helpful to give a background on why CRD updates are potentially risky and some of the nuances involved. Just so someone reading this immediately understands why the webhook is useful, especially if they are not already well-versed in the problem. Could be done as a follow-up though.